charts/opentelemetry-operator/values.yaml

```diff
@@ -220,6 +220,9 @@ admissionWebhooks:
   ## If true and certManager.enabled is false, Helm will automatically create a self-signd cert and secret for you.
   autoGenerateCert: true
 
+  # Set true to reuse existing cert, autoGenerateCert must be enabled too
+  autoGenerateCertIfNotExists: true
```

Comment:
`autoGenerateCertIfNotExists` makes me feel a little confused, and feel separation with existing `autoGenerateCert`, how about put it as a child in `autoGenerateCert` configuration?

Reply:
The following configuration for reference only. If you apply this comments, you may need to change the logic in `operator-webhook.yaml`

```yaml
  ## If true and certManager.enabled is false, Helm will automatically create a self-signd cert and secret for you.
  autoGenerateCert: 
    enabled: true
    # use the old secret  auto generated before, applicable to helm upgrade case.
    reuseExisting: true
```

or 

```yaml
  ## If true and certManager.enabled is false, Helm will automatically create a self-signd cert and secret for you.
  autoGenerateCert: 
    enabled: true
    # true means always re-generate cert
    always: true
```

Reply:
I like your suggestions, I was worried about making breaking changes but if it's acceptable it's the best option. Thanks.

Reply:
If it is consistent with the original logic(re-generate always), I think it is acceptable.